components/brave_private_cdn/private_cdn_helper.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_private_cdn/private_cdn_helper.h"
+
+#include "base/big_endian.h"
+
+namespace brave {
+
+bool PrivateCdnHelper::RemovePadding(std::string* padded_string) const {
+  if (!padded_string) {
+    return false;
+  }
+
+  if (padded_string->size() < 4) {
```

Comment:
seems like a magic number to me; but I'm guessing `RemovePadding` is analogous to the [`decode` Python implementation](https://github.com/brave/simplepadding/blob/master/simplepadding/__init__.py) and that there is a 4 byte header.

Maybe this could be more obvious if there's a constant, like `constexp simple_padding_header_length = 4` and if that is referenced instead? Same with the erase below (0,4). Would be nice to reference the Python implementation too 😄

Reply:
Good point. I can replace those with `sizeof(uint32_t)` since that's where the 4 bytes come from.

Reply:
Fixed in latest revision.